.drone.star

```diff
@@ -3087,35 +2990,3 @@ def uploadTracingResult(ctx):
             ],
         },
     }]
-
-def publishTracingResult(ctx, suite):
```

Comment:
@SwikritiT we cannot post a GitHub comment any more about the tracing result.
But the developers would like to have something in the drone pipeline log that says that a trace is available, and gives a link to where the trace is stored.
Can you adjust something in the drone or other scripts so that it writes useful information to the log?

Reply:
yes I'll do that